lib/cocoapods/command/options/repo_update.rb

```diff
@@ -0,0 +1,34 @@
+module Pod
+  class Command
+    module Options
+      # Provides support for commands to skip updating the spec repositories.
+      #
+      module RepoUpdate
+        module Options
+          def options
+            [
+              ['--no-repo-update', 'Skip running `pod repo update` before install'],
```

Comment:
Another idea would be to generalize that further to `--[no-]repo-update` and interpret both variants here. So users wouldn't to memorize what is allowed per subcommand and could just enforce what they wish. It would also mean that we can easier change the default behavior, because both options are still supported.